network/onchain-discovery/src/test.rs

```diff
@@ -165,7 +165,7 @@ fn read_validator_set(libra_db: &Arc<dyn DbReader>) -> ValidatorSet {
 fn gen_configs(count: usize) -> Vec<NodeConfig> {
     config_builder::ValidatorConfig::new()
         .validators(count)
-        .build_common(true, false)
```

Comment:
makes sense. now that we're just using `LibraDB` directly w/o a network service in between, we don't need to randomize service ports. originally, the problem was that, since unit tests run in parallel, w/o port randomization the different test threads would each try to spin up a storage service on the same port

Reply:
Yes, I think if this were not the case, we would have to figure out how to spread this information. I think getting config/management to replace config-builder is not an overnight project :/